### ISSUE TITLE
{Linked:Bug1098885; Linked:Bug1098894}[Adaptive Cards - Sample - Inputs with validation] - Proper validation error message is not set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'Inputs with validation Sample' page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9309]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Samples' control present on the header section of the page using tab key and activate using enter key.
3. Navigate to the 'Inputs with validation' control present on the left navigation using tab key and activate using enter key.
4. Navigate to the 'Email' and 'Phone' edit field using tab key and enter any invalid data to them.
5. Navigate to the 'Submit' button using tab key and activate using enter key.
6. Verify that, proper validation error message is set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'Inputs with validation Sample' page or not.

**Actual Result:**
Proper validation error message is not set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'Inputs with validation Sample' page. On after activating the 'Submit' button no error message displayed.

**Expected Result:**
A proper validation error message should be displayed and set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'Inputs with validation Sample' page. Additionally, the error message should have a sufficient contrast ratio with their adjacent background color, and the screen reader should properly announce the information regarding it.

**User Experience:**
Users who rely on screen readers or other assistive technologies will be impacted if proper validation error message is not set for the 'Email' and 'Phone' edit field on after filling invalid data present on the 'Inputs with validation Sample' page due to which users are not informed of invalid data entry and might get confusion, and they may unknowingly submit incorrect information.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/error-identification.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/51652ee5-abf5-4037-a727-e04d6e5daa5a

![Image](https://github.com/user-attachments/assets/ca1faf23-3b75-4f67-8425-3c7ed660c8d8)

### Card JSON

```JSON
NA
```
